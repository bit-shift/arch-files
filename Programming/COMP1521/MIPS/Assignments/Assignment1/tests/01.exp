Iteration 0
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
.................@ooooo.................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................































Iteration 1
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
.................ooooo..................
..................@.....................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................































Iteration 2
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
.................oooo...................
.................@o.....................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................































Iteration 3
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
.................ooo....................
.................oo.....................
.................@......................
........................................
........................................
........................................
........................................
........................................
........................................
........................................































Iteration 4
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
.................oo.....................
.................oo.....................
................@o......................
........................................
........................................
........................................
........................................
........................................
........................................
........................................































Iteration 5
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
.................o......................
...............@.oo.....................
................oo......................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
