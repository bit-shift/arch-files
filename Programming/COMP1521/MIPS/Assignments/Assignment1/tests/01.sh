echo 12 | spim -file worm.s 6 5 67
