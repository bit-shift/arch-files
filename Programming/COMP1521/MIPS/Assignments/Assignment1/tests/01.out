Loaded: /usr/local/lib/spim/exceptions.s
[H[2JIteration 0
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
.................@ooooo.................
........................................
........................................
........................................
........................................
........................................
........................................
........................................
